packages/client/src/views/SysAdmin/Performance/WorkflowStatus.tsx

```diff
@@ -529,7 +532,9 @@ function WorkflowStatusComponent(props: WorkflowStatusProps) {
             (eventProgress.registration &&
               eventProgress.registration.status &&
               intl.formatMessage(
-                StatusMapping[eventProgress.registration.status].labelDescriptor
+                StatusMapping[
+                  eventProgress.registration.status as keyof IStatusMapping
+                ]?.labelDescriptor
```

Comment:
```suggestion
                ].labelDescriptor
```
Now we should know all the statuses are handled